src/hotspot/cpu/aarch64/jniFastGetField_aarch64.cpp

```diff
@@ -32,7 +32,7 @@
 #include "prims/jvm_misc.hpp"
 #include "prims/jvmtiExport.hpp"
 #include "runtime/safepoint.hpp"
-#include "runtime/threadWXSetters.inline.hpp"
+#include "runtime/thread.inline.hpp"
```

Comment:
I still don't see why this is needed.

Reply:
I must have forgotten this line during refactoring.